concepticondata/conceptlists/Huttenlocher-1979-225.tsv

```diff
@@ -0,0 +1,226 @@
+Huttenlocher-1979-225-10	10	10	cat	noun	animals	1.A	146	1208	CAT
+Huttenlocher-1979-225-11	11	11	mouth	noun	body parts	1.A	146	674	MOUTH
+Huttenlocher-1979-225-12	12	12	door	noun	grooming	1.A	146	1567	DOOR
+Huttenlocher-1979-225-13	13	13	shoe	noun	clothing	1.A	146	1381	SHOE
+Huttenlocher-1979-225-14	14	14	sheep	noun	animals	1.A	146	1331	SHEEP
+Huttenlocher-1979-225-15	15	15	hand	noun	body parts	1.A	146	1277	HAND
+Huttenlocher-1979-225-16	16	16	bed	noun	grooming	1.A	146	1663	BED
+Huttenlocher-1979-225-17	17	17	pants	noun	clothing	1.A	146	809	TROUSERS
+Huttenlocher-1979-225-18	18	18	cow	noun	animals	1.A	146	1007	COW
+Huttenlocher-1979-225-19	19	19	nose	noun	body parts	1.A	146	1221	NOSE
+Huttenlocher-1979-225-20	20	20	desk	noun	grooming	1.A	146	1775	TABLE
+Huttenlocher-1979-225-21	21	21	shirt	noun	clothing	1.A	146	1622	SHIRT
+Huttenlocher-1979-225-22	22	22	duck	noun	animals	1.A	146	1360	DUCK
+Huttenlocher-1979-225-23	23	23	ear	noun	body parts	1.A	146	1247	EAR
+Huttenlocher-1979-225-24	24	24	house	noun	grooming	1.A	146	1252	HOUSE
+Huttenlocher-1979-225-25	25	1	walk	verb	verbs of movement	1.B	146	1443	WALK
+Huttenlocher-1979-225-26	26	2	remain	verb	stative verbs	1.B	146	987	REMAIN
+Huttenlocher-1979-225-27	27	3	say	verb	verbs of saying	1.B	146	1458	SAY
+Huttenlocher-1979-225-28	28	4	skip	verb	verbs of movement	1.B	146		
+Huttenlocher-1979-225-29	29	5	sleep	verb	stative verbs	1.B	146	1585	SLEEP
+Huttenlocher-1979-225-30	30	6	report	verb	verbs of saying	1.B	146		
+Huttenlocher-1979-225-31	31	7	trot	verb	verbs of movement	1.B	146		
+Huttenlocher-1979-225-32	32	8	wait	verb	stative verbs	1.B	146	1590	WAIT (FOR)
+Huttenlocher-1979-225-33	33	9	tell	verb	verbs of saying	1.B	146	1711	TELL
+Huttenlocher-1979-225-34	34	10	jump	verb	verbs of movement	1.B	146	1829	JUMP
+Huttenlocher-1979-225-35	35	11	sit	verb	stative verbs	1.B	146	1416	SIT
+Huttenlocher-1979-225-36	36	12	question	verb	verbs of saying	1.B	146	1309	ASK (INQUIRE)
+Huttenlocher-1979-225-37	37	13	run	verb	verbs of movement	1.B	146	1519	RUN
+Huttenlocher-1979-225-38	38	14	relax	verb	stative verbs	1.B	146	168	REST
+Huttenlocher-1979-225-39	39	15	call	verb	verbs of saying	1.B	146	1084	CALL
+Huttenlocher-1979-225-40	40	16	hike	verb	verbs of movement	1.B	146		
+Huttenlocher-1979-225-41	41	17	stay	verb	stative verbs	1.B	146	987	REMAIN
+Huttenlocher-1979-225-42	42	18	speak	verb	verbs of saying	1.B	146	1623	SPEAK
+Huttenlocher-1979-225-43	43	19	hop	verb	verbs of movement	1.B	146		
+Huttenlocher-1979-225-44	44	20	rest	verb	stative verbs	1.B	146	168	REST
+Huttenlocher-1979-225-45	45	21	ask	verb	verbs of saying	1.B	146	1309	ASK (INQUIRE)
+Huttenlocher-1979-225-46	46	22	dance	verb	verbs of movement	1.B	146	3472	DANCE (MOVEMENT)
+Huttenlocher-1979-225-47	47	23	lounge	verb	stative verbs	1.B	146		
+Huttenlocher-1979-225-48	48	24	talk	verb	verbs of saying	1.B	146	2708	TALK
+Huttenlocher-1979-225-49	49	25	jog	verb	verbs of movement	1.B	146	2899	JOG (SOMETHING)
```

Comment:
I assume here 'jog' is meant in sense of relaxed running while JOG (SOMETHING) refers to the act of shaking.